modules/special-rules/pages/a-dying-race.adoc

```diff
@@ -0,0 +1,16 @@
+= A Dying Race
+
+Every Eldar casualty is a tragedy, one step closer to the race's extinction.
+
+---
+
+Apply these rules to detachments with the {a-dying-race} ability:
+
+* If the detachment is wiped out, immediately remove D3 points from the player's army morale. [either/or]
+* If the detachment is wiped out, the player's army morale immediately goes down by D3 points. [either/or]
+
++[TODO: decide on the syntax for army morale changes. The original text used several different options, I suggest 'goes up by X points' and 'goes down by X points'.]
```

Comment:
```suggestion
```
I agree with your general idea, but never in passive case. The {active case, second person, imperative} approach is almost always better.

Reply:
OK, but you need to give me a version that works with {active case, second person, imperative}.

Maybe 'raise/lower'? For example:

> If the detachment is wiped out, immediately lower the player's army morale by D3 points.

Reply:
I did: "If the detachment is destroyed, remove D3 points from your army morale."
Technically, "is destroyed" is still passive case, but this is one of the exceptions where passive case is actually better.
No need for raise/lower, add/remove (or perhaps add/deduct) is more plain, without the figurative implication.

Reply:
For what it's worth, the figurative implication is the whole point of suggesting raise/lower. There might not be a 'need' for them, but in some cases I find that plainer text is less appropriate as it can disguise the implications of what's happening by making the text too impersonal.